File: linear_search/linear_search.cpp
#include<iostream>
#include<conio.h>
#include<array>

using namespace std;

int search_linear(int arr[], int element, int c)
{
    int e = element;
    for(int a= 0; a<c; a++)
    {
        if(arr[a] == e )
        {
            return a;
        }
    }
    return 1;
}
int main()
{
    int element = 1000;
    int arr[] = {10, 100, 100, 100, 100000, 1000, 1} ;
    int c = end(arr) - begin(arr) ;
    int s_linear = search_linear(arr, element, c);
    if(s_linear == 1)
    {
        cout<<"element not found";
    }
    else if(s_linear != 1)
    {
        cout<<"Element found at index "<<s_linear ;
    }

    return 0;

}

File: linear_search/linear_search.java
public class Linear_Search
{
    //    tbs = to be searched
    public static int algo(int tbs, int ar[]){
        for(int a = 0; a < (ar.length); a++){
            if(ar[a] == tbs) {
                return a;
            }
        }
        return -1;
    }
    public static void main(String args[]){
        int arr[] = {30, 19, 75, 87, 65, 98, 1456, 1} ;
        int tbs = 65 ;
        int found = algo(tbs, arr) ;
        if (found == -1){
            System.out.print("Element is not found !!") ;
        }
        else if( found != -1){
            System.out.print("Element is found at index " + found) ;
        }

    }
}